definitions/linux.yaml

```diff
@@ -137,6 +137,116 @@ labels: [Authentication, Configuration Files]
 provides: [users.username]
 supported_os: [Linux]
 ---
+name: ProcMounts
+doc: Current mounted filesystems.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/mounts'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysFsSecurity
+doc: Filesystem hardening settings.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/fs/protected_hardlinks'
+      - '/proc/sys/fs/protected_symlinks'
+      - '/proc/sys/fs/suid_dumpable'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysKernelBootloader
+doc: Bootloader state acquired from the kernel.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/bootloader_type'
+      - '/proc/sys/kernel/bootloader_version'
+labels: [System]
+supported_os: [Linux]
+---
+name: ProcSysKernelMemoryPrediction
+doc: Memory address obfuscation settings.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/proc/sys/kernel/dmesg_restrict'
```

Comment:
from a quick read these are only very loosely related.  I think all of these will be much more generally useful if we break them down individually with better names.  I can guess what dmesg_restrict is about, but if it was in it's own artifact called "DmesgReadPrivilegesRestriction" I wouldn't have to.  It would also be good to have canonical URL references for each of these so I don't have to go searching.